contrib/sarplus/scala/python/setup.py

```diff
@@ -0,0 +1,10 @@
+from distutils.core import setup
```

Comment:
not sure i understand why scala/python directory and this setup.py exist?

Reply:
this is not just a scala package, but a Spark package. That way I can automatically pull in the python dependency. Notice in the Jupyter notebook that there is no need to do a "pip install pysarplus" ?